hip-more-cultural-names/common/landed_titles/zzz_HIP_MoreCulturalNames.txt

```diff
@@ -2140,12 +2140,6 @@ e_hre = {
                 }
             }
         }
-        d_osnabruck = {
-            roman = "Asenbruggensis"
```

Comment:
Just wanted to make sure: Are these removals due to the new HIP / SWMH version introducing their own / same variants for these names?

Reply:
they got removed

Reply:
HIP will maybe get released tomorrow